### PR TITLE
modify the path for stream

### PR DESCRIPTION
**Pull request by YuiKasagi**
This may solve a problem that occurred when setting different paths were set up for `datadir` and `anadir`.
Please try `IRD_stream.py` with setting the arbitrary path for `anadir`.

### CODE REVIEW

File: examples/python/IRD_stream.py [hunk 1]
```diff
@@ -76,36 +76,39 @@
...
-flat.dispcor()
+flat.dispcor(master_path=thar.anadir)
 
 # combine & normalize
-target.normalize1D(flatid=flat.streamid)
+target.normalize1D(flatid=flat.streamid,master_path=flat.anadir)
 
-"""### mmfmmf (test) ###
+"""
+### FOR RV MEASUREMENTS ###
```
Comment (YuiKasagi) [inline examples/python/IRD_stream.py:93]: The commented-out part below is the part that creates some files needed for RV measurements. So, you can ignore the change.

**Review by HajimeKawahara — APPROVED** [review]
All of the tests passed. Thanks.

File: src/pyird/utils/irdstream.py [hunk 2]
```diff
@@ -201,7 +201,7 @@ def flatten(self, trace_path=None, extout='_fl', extin=None, mask=None):
         if not self.imcomb:
             extin_noexist, extout_noexist = self.check_existence(extin, extout)
             for i, fitsid in enumerate(tqdm.tqdm(extin_noexist)):
-                filen = self.rawdir/extin_noexist[i]
+                filen = self.anadir/extin_noexist[i]
```
Comment (YuiKasagi) [inline src/pyird/utils/irdstream.py:204]: I noticed that `anadir` is better here.
(The additional modification for this PR is only this.)

**Review by kemasuda — APPROVED** [review]
Thanks a lot for the modification! All the tests worked, and I was also able to use IRD_stream.py to put all the output files into a single common directory!

**Pull request merged**